src/HasActivation.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Interface HasActivation.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP;
+
+/**
+ * An service that needs to run logic during plugin activation.
+ *
+ * @package AmpProject\AmpWP
+ */
+interface HasActivation {
```

Comment:
Note this interface is not currently used.

Reply:
It's not being implemented, but it's used in `\AmpProject\AmpWP\Services::class`:

https://github.com/ampproject/amp-wp/blob/08506c2bc6ff0e8f67d158315e4a8e4767e7e3a0/src/Services.php#L58